provisioner/powershell/provisioner.go

```diff
@@ -335,18 +335,18 @@ func (p *Provisioner) Provision(ctx context.Context, ui packer.Ui, comm packer.C
 		return nil
 	}
 
-	err := retry.Config{StartTimeout: p.config.StartRetryTimeout}.Run(ctx, func(ctx context.Context) error {
+	err := retry.Config{StartTimeout: time.Minute, RetryDelay: func() time.Duration { return 10 * time.Second }}.Run(ctx, func(ctx context.Context) error {
```

Comment:
Nice ! I was worried about changing the default of `p.config.StartRetryTimeout` but this is retrying the cleanup phase which is something else !